test/test_torch.py

```diff
@@ -189,6 +189,7 @@ def test_namespace(ns, *skips):
                        'is_coalesced',
                        'is_distributed',
                        'is_floating_point',
+                       'is_complex',
```

Comment:
is it possible to actually test this functionality?

Reply:
I can test that everything returns false :) (I can't test the true case because it's not currently possible to create complex tensors in PyTorch).